tests/test-rust-setup.el

```diff
@@ -44,91 +44,101 @@
 (defun build-script-crate-file (file-name)
   (expand-file-name file-name "tests/build-script-test"))
 
+(defun cdrassoc (sym alist) (cdr (assoc sym alist)))
```

Comment:
You can use `alist-get` here.  It's in Emacs 26.1 only, but for the tests I think it's fine.

Reply:
Ah wait, we are testing with Emacs 24.5 on Travis, so probably not a good idea.

Reply:
Backporting and feature-detecting is probably overkill for the tests.  So let's leave it as is.  `cdrassoc` is fine.